ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_nested_callbacks/README.rst

```diff
@@ -0,0 +1,46 @@
+
+Nested callbacks example.
+=========================
+
+This example shows a ``Span`` for a top-level operation, and how it can be passed down on a list of nested callbacks (always one at a time), have it as the active one for each of them, and finished **only** when the last one executes. For Python, we have decided to do it in a **fire-and-forget** fashion.
+
+Implementation details:
+
+
+* For ```asyncio`` and ``threading``\ , the ``Span`` is manually passed down the call chain, activating it in each corotuine/task.
```

Comment:
it doesn't look like the span is activated in child coroutines, but rather referenced directly in the closure. Also, I believe with Contextvars we shouldn't need to re-active the span in children.

Reply:
Same as before (rusty comment, etc).

Reply:
You're right, for asyncio and tornado it is not needed to activate.